src/spec/shared/shared_support/mysql.rb

```
@@ -61,7 +61,7 @@ def execute_sql(sql, this_db_name = db_name)
     end
 
     def sql_results_for(sql, this_db_name = db_name)
-      %x{#{sql_cmd(sql, this_db_name)} 2> /dev/null}.lines.to_a[1..-1] || []
+      %x{#{sql_cmd(sql, this_db_name)} -N 2> /dev/null}.lines.to_a || []
```

RuboCop (1.88.2)

[warning] `[]` will never evaluate because `%x{#{sql_cmd(sql, this_db_name)} -N 2> /dev/null}.lines.to_a` always returns a truthy value.

(Lint/UselessOr)